### PR TITLE
Removes conformance validations

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-899

### What?

I have added/removed/altered:

- [x] Removes conformance plugin and all related specs, functionality and email support
- [x] Linted some of the files I've touched as much as I can. I didn't want to make this PR even more complicated that it already was so have forgone changing all of the files I've touched.

### Why?

I am doing this because:

- We aren't the source of truth of the data and have these turned off anyway/haven't been running them in any environment for a very long time

### AC

- [ ] Ran all of Madhu's e2e tests
- [x] Validated that admin integration with updates controller still working
- [x] Validated that frontend fetching of latest updates still works
- [x] Leaves the related tables/data alone so we can easily rollback this PR

### CODE REVIEW

**Review by jebw — COMMENTED** [review]
Seems ok - the change to explicitly require a ruby file inside a 'find_` method looks unusual and is a change of behaviour?

The bit I've not been able to determine is whether validation is disabled? Is obviously disabled from the rake task - but TaricUpdate uses TaricImporter with validation enabled.

I've been trying to trace through how the updates system works - and I've a suspicion conformance validations are still being run - from `TaricUpdate.sync` ? Are you sure they're not currently running?

File: lib/sequel/plugins/nullable.rb [hunk 1]
```diff
@@ -3,10 +3,13 @@ module Plugins
     module Nullable
       module DatasetMethods
         def first_or_null(*args)
-          first(*args) || "Null#{model}".constantize.new
+          null_model = "Null#{model}"
+
+          require null_model.underscore
```
[the remark this answers is not in the document]
Comment (willfish) [reply lib/sequel/plugins/nullable.rb:8]: This was me debugging a failing spec. Have fixed now. Will remove

**Comment by willfish** [comment]
So as far as I know these are just plain not used in CDS imports and don't really ever get triggered/used/add value in Taric @jebw

**Review by jebw — APPROVED** [review]
LGTM

**Pull request merged**